src/person.rb

```diff
@@ -12,6 +12,7 @@ def initialize(age, name: 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
```

Comment:
- [ ] Please, I suggest that a method for adding rentals should be present since there should be a `has-many` side of  Person and Rental (a person has many rentals) and a `belongs-to` side of Rental and Person (a rental belongs to a person). :pray: